src/internal/_sortedKeys.js

```diff
@@ -0,0 +1,7 @@
+var _curry1 = require('./_curry1');
+var _keys = require('./_keys');
+
+
+module.exports = _curry1(function(obj) {
```

Comment:
Ahh, because this was originally the public `keys` function until I realized that I wanted to make a point of being explicit in calling sort.  I'll move this to the public function, and then my caveat above is no longer necessary.  It'll have to wait until I'm on a computer, though.